custom-connectors/EleadProductReference/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Elead Product Reference Data",
+    "version": "1.0",
+    "description": "This API provides data to be used in conjunction with other eLeads APIs as needed.API searches, retrieves and provides the following details - Opportunity sources and sub sources for a specified company and up types, opportunity statuses for a specified company, sales steps for a specified company, employee positions for a specified company, employees by company and position, sender email addresses for a specified company, available vehicle class, year, make, model, trim.",
```

Comment:
There is a missing space after `other eLeads APIs as needed` and the following sentence of `API searches, retrieves and provides the following`